### PR TITLE
Fire/Pipe Fix

### PR DESCRIPTION
**Pull request by Technus**

### CODE REVIEW

**Comment by Technus** [comment]
Has some commits from alkalus on top of it.

File: src/Java/gtPlusPlus/xmod/gregtech/registration/gregtech/GregtechHiAmpTransformer.java [hunk 1]
```diff
@@ -54,7 +56,7 @@ public static void run(){
 			} catch (Throwable t){	
 				mItem_3 = ItemList.Circuit_Ultimate.get(1);
 			}
-		
+		if(!GTNH){
```
Comment (draknyte1) [inline src/Java/gtPlusPlus/xmod/gregtech/registration/gregtech/GregtechHiAmpTransformer.java]: Is this meant to be a part of it?

**Comment by Technus** [comment]
wait this was not supposed to go here

**Comment by Technus** [comment]
i assumed github would set the destination branch to the one from which i clicked the create pullreq but nooooo.

If you see this useful for your repo look only at this 2 files.
GregtechMetaPipeEntityFluid.java
GregtechMetaPipeEntity_Cable.java

File: src/Java/gtPlusPlus/xmod/gregtech/registration/gregtech/GregtechHiAmpTransformer.java (hunk 1 above)
Comment (Technus) [reply src/Java/gtPlusPlus/xmod/gregtech/registration/gregtech/GregtechHiAmpTransformer.java]: Clarified below.

**Pull request closed without merging**